doc/signal_path.rst

```diff
@@ -0,0 +1,302 @@
+   \draw[->, dotted] (delays) to[lbl, auto, edge label=f32] (eqx);
+   \draw[->, dotted] (delays) to[lbl, auto, edge label=i32] (cdelayy);
+   \draw[->, dotted] (delays) to[lbl, auto, edge label'=f32] (eqy);
+   \draw[->, dotted] (eq) to[lbl, auto, edge label'=cf32] (eqx);
+   \draw[->, dotted] (eq) to[lbl, auto, edge label=cf32] (eqy);
+   \draw[->] (quantx) |- node[lbl, auto, swap, near start] {ci8} (pack);
+   \draw[->] (quanty) |- node[lbl, auto, near start] {ci8} (pack);
+
+Delay
+^^^^^
+Delays may be specified with sub-sample precision. To handle this, the delay
+is split into two components: a :dfn:`coarse` delay (a whole number of
+samples) and a :dfn:`fine` delay (between -0.5 and +0.5 samples). The coarse
+delay is applied as a shift in time, while the fine delay is applied as a
+phase slope in the frequency domain. As noted in :ref:`math-delay`, the user
+provides the overall phase adjustment for the centre frequency, and the
+constant term of the phase slope is computed from that (taking into account
+the effect of the coarse delay on phase).
+
+The fine delay and the fixed phase offset for each spectrum are computed in
+double precision then reduced to single precision for application. Conversion
+of the delay to a per-channel phase correction, and of phases to complex
+phasors are done in single precision.
+
+Polyphase filter bank (PFB)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A finite impulse response (FIR) filter is applied to the signal to condition
+the frequency-domain response. The filter is the product of a Hann window (to
+reduce spectral leakage) and a sinc (to broaden the peak to cover the
+frequency bin). Specifically, if there are :math:`n` output channels and
+:math:`t` taps in the polyphase filter bank, then the filter has length
+:math:`w = 2nt`, with coefficients
+
+.. math::
+
+   x_i = A\sin^2\left(\frac{\pi i}{w - 1}\right)
+         \operatorname{sinc}\left(w_c\cdot \frac{i + \tfrac 12 - nt}{2n}\right),
+
+where :math:`i` runs from 0 to :math:`w - 1`. Here :math:`A` is a
+normalisation factor which is chosen such that :math:`\sum_i x_i^2 = 1`. This
+ensures that given white Gaussian noise as input, the expected output power
+in a channel is the same as the expected input power in a digitised sample.
+Note that the input and output are treated as integers rather than as
+fixed-point values.
+
+The tuning parameter :math:`w_c` (specified by the :option:`!--w-cutoff`
+command-line option) scales the width of the response in the frequency domain.
+The default value is 1, which makes the width of the response (at -6dB)
+approximately equal the channel spacing.
+
+.. _signal-path.narrow:
+
+Dithering
+^^^^^^^^^
+To improve linearity, a random value selected uniformly from the interval
+(-0.5, 0.5) is added to each component (real and imaginary) before
+quantisation. The random seeds are carefully chosen to ensure that
+random sequences are not shared across antennas.
```

Comment:
I find myself becoming somewhat curious to know how long we'd have to observe in order to see this effect if we did have common dither seeds between antennas.

Reply:
I imagine it would show up quite strongly as a source at the centre of the field (delay centre).